runhouse/main.py

```diff
@@ -246,11 +246,10 @@ def cluster_list(
         help="Time duration to filter on. Minimum allowable filter is 1 minute. You may filter by seconds (s), "
         "minutes (m), hours (h) or days (s). Examples: 30s, 15m, 2h, 3d.",
     ),
-    cluster_status: Optional[ClustersListStatus] = typer.Option(
+    cluster_status: Optional[ClusterStatus] = typer.Option(
         None,
         "--status",
-        help="Cluster status to filter on. Supported filter values: 'running', 'terminated' (cluster is not live), "
-        "'down' (Runhouse server is down, but the cluster might be live).",
+        help="Cluster status to filter on.",
```

Comment:
is it possible to include the supported values directly in typer?

Reply:
the enum is included by default like this:
```
  --status [running|terminated|initializing|unknown]
                                  Cluster status to filter on.
```
does that look ok to you?